app/Utils/utils.php

```diff
@@ -5,6 +5,19 @@
 use Illuminate\Support\Facades\URL;
 use Illuminate\Support\Facades\Redirect;
 
+function full_text_safe($string)
```

Comment:
https://dev.mysql.com/doc/refman/5.7/en/fulltext-boolean.html

Otherwise:
a) we ignore words with "-", "_", "&", "%", "(", ")" etc..
b) MySQL eroor